runtime/ftplugin/debchangelog.vim

```diff
@@ -182,8 +182,8 @@ function <SID>FinaliseMenu()
     amenu enable Changelog.New\ Version
```

Comment:
@adaext Was there a reason this wasn't changed to `&New\ Version`?

Reply:
> @adaext Was there a reason this wasn't changed to `&New\ Version`?

Thanks, I've also fixed it.